proto/rill/runtime/v1/queries.proto

```diff
@@ -279,7 +280,8 @@ message MetricsViewAggregationRequest {
   TimeRange time_range = 12;
   google.protobuf.Timestamp time_start = 6; // Deprecated in favor of time_range
   google.protobuf.Timestamp time_end = 7; // Deprecated in favor of time_range
-  MetricsViewFilter filter = 8;
+  Expression filter = 8;
+  repeated MetricsViewColumnAlias aliases = 13;
```

Comment:
Why are `aliases` needed in other cases than `MetricsViewComparison`? For the other cases, I think it's okay to require clients to use the real names (since there's no ambiguity around base vs. comparison vs. delta values).

Reply:
The idea was to use the same measure in `measures` and `filter`. Aggregation allows for count and count distinct so I thought those could be defined in aliases and used in `measures` and `filter` fields.